README_ko.md

```diff
@@ -156,7 +156,10 @@
 
    ```bash
    $ cd ragflow/docker
+   # CPU
    $ docker compose -f docker-compose.yml up -d
+   # GPU
```

Comment:
Remove these two lines from readme. Same for the rest I checked with the team, the rationale behind this is to keep readme simple and straightforward.